vue2-vuetify/src/dynaform/DateTimeControlRenderer.vue

```diff
@@ -43,64 +67,222 @@ import {
+      if (this.inputFormat == 'date') {
+        this.date = parseDate(
+          'value' in dateFormatted ? dateFormatted.value : dateFormatted
+        );
+      } else if (this.inputFormat == 'time') {
+        this.time = formatTime(
+          'value' in timeFormatted ? timeFormatted.value : timeFormatted
+        );
+      } else if (this.inputFormat == 'date-time') {
+        this.dateTime = parseDateTime(
+          'value' in dateFormatted ? dateFormatted.value : dateFormatted,
+          'value' in timeFormatted ? timeFormatted.value : timeFormatted
+        );
+      }
+    }
+
     return {
       ...useVuetifyControl(useJsonFormsControl(props)),
+      menu: false,
+      date,
+      dateFormatted,
+      time,
+      timeFormatted,
+      dateTime,
+      dateTimeFormatted,
+      formatDate,
+      parseDate,
+      formatTime,
+      formatDateTime,
+      parseDateTime,
+      blurChangeFormat,
     };
   },
+  watch: {
+    date() {
+      this.dateFormatted = this.formatDate(this.date);
+      if (this.inputFormat == 'date-time') {
+        this.dateTimeFormatted = this.formatDateTime(
+          this.date + ' ' + this.time
+        );
+      }
+    },
+    time() {
+      this.timeFormatted = this.formatTime(this.time);
+      if (this.inputFormat == 'date-time') {
+        this.dateTimeFormatted = this.formatDateTime(
+          this.date + ' ' + this.time
+        );
+      }
+    },
+    dateTime() {
+      this.dateTimeFormatted = this.formatDateTime(this.dateTime);
+    },
+  },
   computed: {
     hint(): string {
       return this.control.uischema.options?.hint ?? '';
     },
     placeholder(): string {
       return this.control.uischema.options?.placeholder ?? '';
     },
-    dataTime: {
-      get(): string | null | undefined {
-        const datetimeLocalFormat = 'YYYY-MM-DDTHH:mm:ss.SSS';
-        const saveFormat = this.appliedOptions.dateTimeSaveFormat ?? undefined;
-        const value = this.control.data as string | undefined | null;
+    inputFormat(): string | undefined {
+      return this.control.schema?.format ?? '';
+    },
+    valueChangeFormat(): string | null {
+      if (this.inputFormat == 'date') {
+        return this.dateFormatted;
+      } else if (this.inputFormat == 'time') {
+        return this.timeFormatted;
+      } else if (this.inputFormat == 'date-time') {
+        return this.dateTimeFormatted;
+      } else {
+        return null;
+      }
+    },
+    maxDate(): string {
+      let max = this.control.uischema.options?.maxDate;
+      if (max) {
+        let min = this.control.uischema.options?.minDate;
+        let current = new Date(this.date);
+        max = new Date(max);
+
+        if (current > max) {
+          min ? (min = new Date(min)) : (min = '');
+          if (!min || !(max < min)) {
+            this.date = max.toISOString().split('T')[0];
+          }
+        }
+
+        if (min) {
+          max < min ? (max = min.toISOString()) : (max = max.toISOString());
```

Comment:
change for like this
`max = max < min ?  min.toISOString() : max.toISOString();`